.gitignore

```diff
@@ -163,4 +163,7 @@ tags
 *.lock
 
 # DS_Store (MacOS)
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Lewis
+scratch/
```

Comment:
Not sure if we want to add this in the general gitignore of Transformers?

Reply:
Oop! Will fix that!

Reply:
Fixed :)